.github/workflows/ci.yml

```
@@ -18,32 +23,26 @@ jobs:
+          cache: true
+          cache-dependency-path: |
+            go.sum
+            go.mod
 
       - name: Run Gosec Security Scanner
         # SECURITY: Pinned to specific commit SHA for v2.22.10 (immutable)
         # v2.22.10 supports Go 1.25.3/1.24.9, released Oct 15, 2025
         # Checksum verification: https://github.com/securego/gosec/releases/download/v2.22.10/gosec_2.22.10_checksums.txt
         # GPG signature: https://github.com/securego/gosec/releases/download/v2.22.10/gosec_2.22.10_checksums.txt.gpg
-        uses: securego/gosec@6be2b51fd78feca86af91f5186b7964d76cb1256 # v2.22.10
+        uses: securego/gosec@${{ env.GOSEC_VERSION }} # v2.22.10
```

actionlint (1.7.10)

[error] context "env" is not allowed here. no context is available here. see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)